x-pack/plugins/security_solution/public/management/pages/trusted_apps/store/reducer.ts

```diff
@@ -110,7 +111,7 @@ const trustedAppCreationDialogStarted: CaseReducer<TrustedAppCreationDialogStart
     ...state,
     creationDialog: {
       ...initialCreationDialogState(),
-      formState: { ...action.payload, isValid: true },
+      formState: { ...action.payload, isValid: false },
```

Comment:
out of curiosity, whats this flip for

Reply:
When the create form is initially displayed, we should not assume that content is valid now that the form values are "controlled" via props (a new `trustedApp` prop on `CreateTrustedAppForm`. This was needed to ensure we support the Create flow where the state of the `create` button is controlled via this state value.